main.go

```diff
@@ -93,8 +93,7 @@ func main() {
 
 	namespace, err := getWatchNamespace()
 	if err != nil {
-		setupLog.Error(err, "Failed to get watch namespace")
```

Comment:
I think the operator should expect that the `WATCH_NAMESPACE` env var exists. And fail if it is not there.

Reply:
we reverted this as OLM also handles this